stdlib/Test/src/Test.jl

```diff
@@ -478,18 +484,24 @@ Test Broken
 
 julia> @test 2 + 2 == 4 skip=false
 Test Passed
+
+julia> @test 2 == 4 skip=false
+Test Passed
+
+julia> 
```

Comment:
An example using the `expr` feature might be nice.  e.g. contrast
```jl
for i = 1:2; @test isodd(i); end
```
with
```jl
for i = 1:2; @test isodd(i) expr=:(isodd($i)); end
```

Reply:
Yap, will certainly add something.